webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/content/ActivityLink.vue

```diff
@@ -122,6 +122,19 @@
           @click="displayFullContent">
           <span class="pl-6">{{ $t('UIActivity.label.seeMore') }}</span>
         </v-btn>
+        <div
+          v-if="activityViews"
+          :title="activityViewsTooltip"
```

Comment:
it should be v-tooltip instead IMO as it is done in news detail

Reply:
To follow the guidelines, we use the same navigator tooltip as the activity link title and summary, as confirmed by the PO!